src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/NotFoundEndpointException.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace CoreWCF.Configuration
+{
+    public class NotFoundEndpointException : Exception
+    {
```

Comment:
We already have an EndpointNotFoundException, would it make sense to use that here? In CoreWCF that exception is thrown when a message arrives with a To header for an address that the service isn't configured to accept message for. On WCF client, it's thrown when the server indicates the endpoint doesn't exist, eg with a 404 HTTP response. So it isn't exactly the same scenario, but I don't see an ambiguity with how this would get thrown. I believe it would be different code paths.

Reply:
I think this is semantically correct, there is no reason to create a new type for exception